builtin/fstk/tabview.lua

```diff
@@ -76,7 +76,25 @@ local function get_formspec(self)
 		end
 	end
 
-	local formspec = (prepend or "") .. self:tab_header() .. content
+	local end_button_size = 0.65
```

Comment:
```suggestion
	local end_button_size = 0.75
```

The button is hard to hit on Android, I suggest increasing the size to `0.75`.